src/components/Table.js

```diff
@@ -274,21 +446,41 @@ export default class Table extends React.Component<TableDefaultProps, TableProps
   }
 
   render() {
+    const { multiSelect, selection, activeRowId, anchorRowBackgroundColor } = this.props;
+    const { selectedRowIndices } = this.state;
+
     let selectedRowIds;
-    if (Array.isArray(this.props.selection)) {
-      selectedRowIds = this.props.selection;
-    } else if (typeof this.props.selection === 'string') {
-      selectedRowIds = [this.props.selection];
+    if (Array.isArray(selection)) {
+      selectedRowIds = selection;
+    } else if (typeof selection === 'string') {
+      selectedRowIds = [selection];
     } else {
       selectedRowIds = [];
     }
 
+    // FIXME: Move data manipulation outside of render function.
```

Comment:
Couldn't this be solved by just updating `selectedRowIds` where `selectedRowIndices` is being updated and getting rid of the `selectedRowIndices` set?